docs/resources/dhcp_host.md

```diff
@@ -0,0 +1,82 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_dhcp_host Resource - terraform-provider-bloxone"
+subcategory: "IPAM/DHCP"
+description: |-
+  Manages DHCP Hosts.
+  A DHCP Host object associates a DHCP Config Profile with an on-prem host.
+---
+
+# bloxone_dhcp_host (Resource)
+
+Manages DHCP Hosts.
+
+A DHCP Host object associates a DHCP Config Profile with an on-prem host.
+
+## Example Usage
+
+```terraform
+data "bloxone_infra_hosts" "host" {
+  filters = {
+    display_name = "my-host"
+  }
+}
+
+resource "bloxone_dhcp_server" "dhcp_server" {
+  name = "my-dhcp-server"
+}
+
+resource "bloxone_dhcp_host" "example_dhcp_host" {
+  id     = data.bloxone_infra_hosts.host.results.0.legacy_id
+  server = bloxone_dhcp_server.dhcp_server.id
+
+  # Other Optional fields
+  tags = {
```

Comment:
are you able to update tags through dhcp_host?

Reply:
no it doesnt allow to update tags.

Reply:
can you please remove it from the example then?